cli/flox/src/commands/environment.rs

```diff
@@ -1093,26 +1093,42 @@ impl Update {
     }
 }
 
-/// upgrade packages using their most recent flake
+/// Upgrade packages in an environment
 #[derive(Bpaf, Clone)]
 pub struct Upgrade {
     #[allow(dead_code)] // pending spec for `-e`, `--dir` behaviour
     #[bpaf(external(environment_args), group_help("Environment Options"))]
     environment_args: EnvironmentArgs,
 
-    #[allow(unused)] // Command currently forwarded
     #[bpaf(external(environment_select), fallback(Default::default()))]
     environment: EnvironmentSelect,
 
-    #[allow(dead_code)] // not yet handled in impl
-    #[bpaf(positional("PACKAGES"))]
-    packages: Vec<String>,
+    /// ID of a package or group name to upgrade
+    #[bpaf(positional("package or group"))]
+    groups_or_iids: Vec<String>,
```

Comment:
Do you think that should be added here, or should the option descriptions be pretty short?

Reply:
I think that should be made clear in the command description (not the argument description).

Reply:
Added some command description. Doesn't exactly look lovely...
```
Usage: flox upgrade [-s=SYSTEM] [-d=<path> | -r=<owner/name>] [<package or group>]...

Upgrade packages in an environment
   When no arguments are specified, all packages in the environment are upgraded.
  Packages to upgrade can be specified by either group name or, if a package is not in a group with any
  other packages, it may be specified by ID.
  If the specified argument is both a group name and a package ID, only the group is upgraded.
  Packages without a specified group in the manifest can be upgraded by passing 'toplevel' as the group
  name.
    -s, --system=SYSTEM
    -d, --dir=<path>     Path containing a .flox/ directory
    -r, --remote=<owner/name>  A remote environment on floxhub
    <package or group>   ID of a package or group name to upgrade

Available options:
    -h, --help           Prints help information
```